holoviews/plotting/bokeh/element.py

```diff
@@ -2665,7 +2647,7 @@ class OverlayPlot(GenericOverlayPlot, LegendPlot):
                           'min_height', 'max_height', 'min_width', 'min_height',
                           'margin', 'aspect', 'data_aspect', 'frame_width',
                           'frame_height', 'responsive', 'fontscale', 'subcoordinate_y',
-                          'subcoordinate_scale']
+                          'subcoordinate_scale', 'autorange']
```

Comment:
This is needed to get this to work, have updated the tests to account for this. 

``` python
import holoviews as hv; hv.extension('bokeh')
import numpy as np

np.random.seed(42)
ys = np.random.randn(101).cumsum(axis=0)

hv.Curve(ys) * hv.Curve(ys[::-1]).opts(autorange='y')
```

Reply:
Yes, this looks good, did wonder why I had to set it on the Element and the overlay. Not sure why we hadn't set it to propagate before.

Reply:
Updated again to also work with this one `(hv.Curve(ys) * hv.Curve(ys[::-1])).opts(autorange='y')`. It also makes the range in the multiple-unit test reasonable.